components/VotesList/VotesListItem.tsx

```diff
@@ -210,9 +210,17 @@ export function VotesListItem({
       if (!isCommitted) return "Not committed";
       if (!decryptedVote || !canReveal) {
         if (delegationStatus === "delegator") {
-          return "Delegate Must Reveal";
+          if (isRevealed) {
+            return "Delegate Revealed";
```

Comment:
tiny nit: let's not capitalize each word to stay consistent with the rest of the text